src/utils/tokens/names.js

```diff
@@ -100,3 +101,298 @@ export function useUpdateTokenName() {
     [endpoint],
   );
 }
+// Returns tokenInfos for the popular tokens list.
+export function usePopularTokens() {
+  const tokenInfos = useTokenInfos();
+  const { endpoint } = useConnectionConfig();
+  return (!POPULAR_TOKENS[endpoint]
```

Comment:
Doesn't this defeat the purpose of spl-token-list? As the token mint needs to exist here and there to show...

Reply:
No. This list is only used to decide what tokens to show in the AddTokenDialog. All other tokens that come into the wallet will automatically populate with the correct logo, name, and symbol. In a subsequent PR, we can remove the hard coded list (which is only used for filtering).

Reply:
For further context, the registry is needed as a prerequisite for integrating the wormhole SDK into the wallet, which is the primary motivation and why I'm happy to punt the AddTokenDialog work.